admin/spec/back-end/lib/validator/check-form/single-check-form-validator.spec.js

```diff
@@ -68,6 +68,16 @@ describe('singleCheckFormValidator', function () {
         const singleCheckFormErrors = await singleCheckFormValidator.validate(uploadedFile)
         expect(singleCheckFormErrors).toEqual([`${checkFormErrorMessages.invalidFileCharacters} ${fileName}`])
       })
+      it('and returns a validation error object when two or more additional empty rows are found', async () => {
```

Comment:
why aren't we making this more tolerant and just throwing away the blank lines?

Reply:
indeed

Reply:
@GuyHarwood @jon-shipley The requirement is to validate as false any file that contains extra blank lines.
The validator so far was tolerant by trimming blank lines but that caused issues during submission as blank lines were taken into account.

Reply:
My view is that validating white-space makes it harder for users.

Reply:
i think we should be applying common sense here.  trim the blank lines and don't add them as empty questions. we are potentially making this more complicated for the user otherwise.  Talk to the BAs if you need to

Reply:
the end goal should be to 'keep adding rows with 2 factors until i encounter a row without 2 factors separated by a comma'

Reply:
Business expects a flawless file with no extra blank lines from the test dev submission.
Meaning that additional blank lines shouldn't be allowed anywhere in the file.

Reply:
🤷‍♂️ fair enough